README.md

```diff
@@ -42,6 +42,15 @@ end
 ```
 Set your different credentials in the file by uncommenting the lines and adding your keys.
 
+------------------
+If you are using devise_token_auth gem, you need to run this generator.
+
+    $ rails g apple_sign_in:appple_sign_in_controller [scope]
+
+In the scope yo need to write your path from controllers to yours existents devise controllers.
+An example '$ rails g apple_sign_in:appple_sign_in_controller ap1/v1/'
+This will generate a new controller: `apple_sign_in_controller.rb`.
```

Comment:
```suggestion
This will generate a new controller: `api/v1/apple_sign_in_controller.rb`.
```

Maybe we can write it like this so that the reader knows where it will be copied to